src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -599,30 +600,28 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
         const { openUpgradePlanModal = () => {} } = this.props;
         return (
             <InlineNotice title={<FormattedMessage {...messages.upgradeInlineNoticeTitle} />} type="info">
-                <FormattedMessage
-                    values={{
-                        upgradeGetMoreAccessControlsLink: (
-                            <PlainButton
-                                className="upgrade-link"
-                                data-resin-target="external_collab_top_message_upgrade_cta"
-                                onClick={openUpgradePlanModal}
-                                type="button"
-                            >
-                                <FormattedMessage {...messages.upgradeLink} />
-                            </PlainButton>
-                        ),
-                        collaboratorAccessLink: (
-                            <Link
-                                className="upgrade-link"
-                                href="https://support.box.com/hc/en-us/articles/360044196413-Understanding-Collaborator-Permission-Levels"
-                                target="_blank"
-                            >
-                                <FormattedMessage {...messages.collaboratorAccessLink} />
-                            </Link>
-                        ),
-                    }}
-                    {...messages.upgradeCollaboratorAccessDescription}
-                />
+                <FormattedCompMessage
+                    id="boxui.unifiedShare.upgradeCollaboratorAccessDescription"
```

Comment:
Shouldn't this generate a line in `i18n/en-US.properties` to ensure it gets exported for translation?

Reply:
No because the Box webpack plugin that extracts it is not open-source, so we cannot put it in the build scripts for BUIE. Instead, we have a special build in the globalization jenkins job which augments the BUIE build and installed this private webpack plugin. That extracts the string and gets it translated through the normal route and finally checks it all in to the translated properties files.